app/src/main/java/com/kickstarter/services/interceptors/GraphQLInterceptor.kt

```diff
@@ -1,17 +1,22 @@
 package com.kickstarter.services.interceptors
 
+import com.kickstarter.libs.Build
 import com.kickstarter.libs.CurrentUserType
 import okhttp3.Interceptor
 import okhttp3.Response
 
-class GraphQLInterceptor(private val clientId: String, private val currentUser: CurrentUserType) : Interceptor {
+class GraphQLInterceptor(private val clientId: String,
+                         private val currentUser: CurrentUserType,
+                         private val build: Build) : Interceptor {
     override fun intercept(chain: Interceptor.Chain?): Response {
         val original = chain!!.request()
         val builder = original.newBuilder().method(original.method(), original.body())
-        if (this.currentUser.exists()) {
-            builder.addHeader("Authorization", "token " + this.currentUser.accessToken)
+        val accessToken = this.currentUser.accessToken
```

Comment:
We were checking for existence of `this.currentUser` previously, so could we run into a null-pointer exception by keying into `.accessToken`?

Reply:
i don't think so because `currentUser` is just an interface. and calling `.accessToken` is fetching a stored property 🤔 but i don't want to introduce anything so i'm going to revert it!